File: 18-08-2023 1943/1943.c
#include <stdio.h>
 
int main() {
 
    int k;
    
    scanf("%d", &k);
    
    if (k > 50) {
        printf("Top 100\n");
    }
    if (k <= 50 && k > 25) {
        printf("Top 50\n");
    }
    if (k <= 25 && k > 10) {
        printf("Top 25\n");
    }
    if (k <= 10 && k > 5) {
        printf("Top 10\n");
    }
    if (k <= 5 && k > 3) {
        printf("Top 5\n");
    }
    if (k <= 3 && k > 1) {
        printf("Top 3\n");
    }
    if (k == 1) {
        printf("Top 1\n");
    }
 
    return 0;
}

File: AD-HOC/1943/1943.py
top = int(input())

if top > 50:
    print("Top 100")
elif top > 25:
    print("Top 50")
elif top > 10:
    print("Top 25")
elif top > 5:
    print("Top 10")
elif top > 3:
    print("Top 5")
elif top > 1:
    print("Top 3")
elif top == 1:
    print("Top 1")